recipes/ux.rb

```diff
@@ -36,62 +36,53 @@
 end
 
 log 'Checkout and Unzip the latest UX Code'
-remote_file '/home/jellyfish/ux-master.zip' do
+remote_file "#{node['rbenv']['user_home']}/ux-master.zip" do
   source 'https://github.com/projectjellyfish/ux/archive/master.zip'
   mode '0644'
 end
 
 log 'Unzip ux-master and move it to ux'
 bash 'unzip ux-master.zip' do
-  cwd '/home/jellyfish'
-  user 'jellyfish'
+  cwd node['rbenv']['user_home']
+  user node['jellyfish']['user']
   code <<-EOH
-  unzip ux-master.zip
+  /usr/bin/unzip ux-master.zip; mv ux-master ux
```

Comment:
Though mv and cp were shell built-ins. Just looked and they are not. Will add full path /bin/mv